src/promptflow-core/promptflow/_utils/flow_utils.py

```diff
@@ -321,3 +323,157 @@ def parse_variant(variant: str) -> Tuple[str, str]:
             message=str(error),
             error=error,
         )
+
+
+def format_signature_type(flow_meta):
```

Comment:
may put all signature related into another file

Reply:
shall this be in devkit package?

Reply:
moved to devkit